{"name": "search on google", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\n  (Session info: headless chrome=101.0.4951.54)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x00D6B8F3+2406643]\n\tOrdinal0 [0x00CFAF31+1945393]\n\tOrdinal0 [0x00BEC610+837136]\n\tOrdinal0 [0x00C149C3+1001923]\n\tOrdinal0 [0x00C140A5+999589]\n\tOrdinal0 [0x00C3427C+1131132]\n\tOrdinal0 [0x00C0FA64+981604]\n\tOrdinal0 [0x00C34494+1131668]\n\tOrdinal0 [0x00C44682+1197698]\n\tOrdinal0 [0x00C34096+1130646]\n\tOrdinal0 [0x00C0E636+976438]\n\tOrdinal0 [0x00C0F546+980294]\n\tGetHandleVerifier [0x00FD9612+2498066]\n\tGetHandleVerifier [0x00FCC920+2445600]\n\tGetHandleVerifier [0x00E04F2A+579370]\n\tGetHandleVerifier [0x00E03D36+574774]\n\tOrdinal0 [0x00D01C0B+1973259]\n\tOrdinal0 [0x00D06688+1992328]\n\tOrdinal0 [0x00D06775+1992565]\n\tOrdinal0 [0x00D0F8D1+2029777]\n\tBaseThreadInitThunk [0x7544FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77517A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77517A4E+238]\n\n", "trace": "  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\qademo.py\", line 20, in step_impl\n    searchBar.send_keys(\"list of cloud platform\")\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 570, in send_keys\n    self._execute(Command.SEND_KEYS_TO_ELEMENT,\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 740, in _execute\n    return self._parent.execute(command, params)\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 430, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given search keyword on google", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.ElementNotInteractableException: Message: element not interactable\n  (Session info: headless chrome=101.0.4951.54)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x00D6B8F3+2406643]\n\tOrdinal0 [0x00CFAF31+1945393]\n\tOrdinal0 [0x00BEC610+837136]\n\tOrdinal0 [0x00C149C3+1001923]\n\tOrdinal0 [0x00C140A5+999589]\n\tOrdinal0 [0x00C3427C+1131132]\n\tOrdinal0 [0x00C0FA64+981604]\n\tOrdinal0 [0x00C34494+1131668]\n\tOrdinal0 [0x00C44682+1197698]\n\tOrdinal0 [0x00C34096+1130646]\n\tOrdinal0 [0x00C0E636+976438]\n\tOrdinal0 [0x00C0F546+980294]\n\tGetHandleVerifier [0x00FD9612+2498066]\n\tGetHandleVerifier [0x00FCC920+2445600]\n\tGetHandleVerifier [0x00E04F2A+579370]\n\tGetHandleVerifier [0x00E03D36+574774]\n\tOrdinal0 [0x00D01C0B+1973259]\n\tOrdinal0 [0x00D06688+1992328]\n\tOrdinal0 [0x00D06775+1992565]\n\tOrdinal0 [0x00D0F8D1+2029777]\n\tBaseThreadInitThunk [0x7544FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x77517A7E+286]\n\tRtlGetAppContainerNamedObjectPath [0x77517A4E+238]\n\n", "trace": "  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\qademo.py\", line 20, in step_impl\n    searchBar.send_keys(\"list of cloud platform\")\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 570, in send_keys\n    self._execute(Command.SEND_KEYS_TO_ELEMENT,\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py\", line 740, in _execute\n    return self._parent.execute(command, params)\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 430, in execute\n    self.error_handler.check_response(response)\n  File \"D:\\Program Files\\python\\python3104\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 247, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1652050333752, "stop": 1652050337156}, {"name": "Then we should get result on ZDnet", "status": "skipped", "start": 1652050337157, "stop": 1652050337157}], "start": 1652050333752, "stop": 1652050337157, "uuid": "b8c2bf13-d982-48a4-9e4a-7564b95b38ab", "historyId": "bad2b9691863e6dee9292ef10ad5f9b5", "fullName": "QA Demo: search on google", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "QA Demo"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}